frontend/src/lib/components/ServiceRequestDetailsForm.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
```

Comment:
Is it going to be done in a subsequent story and if so did you create a ticket for it?

Reply:
Forgot I set this up originally. Meant to delete it and it slipped my mind. Will delete